docs/user-guide/cli-using-initializing-user-configuration.md

```diff
@@ -0,0 +1,68 @@
+# Initializing user configuration
+
+As an application developer or Zowe CLI user, you can manage your connection details efficiently and in one location.
+
+That could mean relying on a team configuration file, or creating your own *user* configuration file.
+
+To create your own user configuration, start with a global team configuration file that you have created or was provided to you. In this way, a single global configuration can become the basis for multiple user-specific configurations that are created with modifications tailored to particular requirements.
+
+To learn more about how profiles and configuration files work, see [How Zowe CLI uses configurations](../user-guide/cli-using-understand-profiles-configs.md).
+
+
+**[is there a use case for a team leader to create a user config file for *someone else*? We address app developers in the first graf. But I also want to make sure that we don't need to add instructions or best practices or context in this top section for team leaders.]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
It could be useful for a team leader to also distribute user configs if there are certain profile properties that might vary between users (and it might be clearer which properties should be updated if they are stored in a separate user config), but I don't really see much benefit in providing a user config since the same changes could be made in a team config. It's possible that there is something I'm missing.